### DISCUSSION TITLE
How do you open a vault on iOS?

### DISCUSSION THREAD

**Discussion by hollyhudson** [remotely-save/remotely-save#775] [General]
I have the plugin set up in a vault on Nextcloud on my Mac, checked the connectivity and it reports it's working, but now how do I open the vault in Obsidian on my iPad? The files are on my iPad (I've made sure Nextcloud has them downloaded) but the vault doesn't show up as an option in Obsidian.

**Comment by fyears** [comment]
i don't get it. do you mean you install both nextcloud app and obsidian app on ipad?

**Comment by fyears** [comment]
the plugin doesn't work like that. you don't need to install nextcloud app on ipad.

just install obsidian and create an empty local vault using the same name as mac, then install remotely save and configure it, then the contents should be downloaded from online nextcloud website to local obsidian vault

**Reply by hollyhudson** [reply]
Nextcloud has been installed on my iPad for years because I use it daily for lots of other things. I just would like to move my Obsidian notes off of Apple's servers and onto my own Nextcloud server.  Unfortunately, Obsidian won't give me the option to open a vault that's in Nextcloud, so I'm trying to find a workaround.

I'm wondering if creating an empty vault and letting remotely-save populate it is going to result in me having the vault duplicated on my devices. That's not going to work for me because of the size of the vault I want to sync. And I do also need the files available in Nextcloud because sometimes I edit them directly (especially non-markdown files like .csv files) in other apps on my iPad. It's frustrating that I can't have Obsidian just look at the folder containing my vault that is already sitting there on my iPad.

**Reply by fyears** [reply]
sorry but usually on ios the apps (and their contents) are isolated.

i understand your concerns but remotely save doesn't work by acknowledging other apps on the same devices. i wish you can find a workaround
